### PR TITLE
[BACKPORT] Hotfix construction base

### PR DESCRIPTION
**Pull request by szabo137**
backport of the hotfix #84

### CODE REVIEW

**Review by AntonReinhard — APPROVED** [review]

File: Project.toml [hunk 1]
```diff
@@ -7,7 +7,7 @@ authors = [
     "Tom Jungnickel",
     "Anton Reinhard",
 ]
-version = "0.2.0"
+version = "0.2.1"
 
 [deps]
```
Comment (AntonReinhard) [inline Project.toml:12]: Doesn't `ConstructionBase` have to be added here too?

**Comment by AntonReinhard** [comment]
I think the rebasing was messed up here, maybe you could just reset the branch to dev, cherry-pick commit 16472ce93235927f513865b1e044b91fa2218b32 and then force push to this again? That might be cleaner and should fix the ConstructionBase problem.

File: Project.toml (hunk 1 above)
Comment (szabo137) [reply Project.toml:12]: Now it is.

Comment (AntonReinhard) [reply Project.toml:12]: It's still not here, I think you cherry-picked onto an outdated dev branch :see_no_evil:

Comment (szabo137) [reply Project.toml:12]: Sorry, my bad, sometimes I am too stupid for this. Thanks for taking care!

**Review by AntonReinhard — APPROVED** [review]
Should be correct now

**Comment by AntonReinhard** [comment]
@szabo137 please approve and merge

**Comment by szabo137** [comment]
> @szabo137 please approve and merge

can't approve this, because I am the creator of this PR. I think you can't either because you pushed to the PR. So, I requested @SimeonEhrig to hit the approve button for us 😅

**Review by SimeonEhrig — APPROVED** [review]

**Pull request merged**